app/src/main/java/com/hamdan/forzenbook/view/ForzenbookActivity.kt

```diff
@@ -125,4 +142,28 @@ class ForzenbookActivity : ComponentActivity() {
             }
         }
     }
+
+    companion object {
+        val LOGIN_PAGE =
```

Comment:
Why the ones with errors?

Reply:
in case they woul want to be used in the future for something, I can remove them for now if you would like

Reply:
removed the ones with errors